pkg/session_recording/recordings.jsx

```diff
@@ -425,13 +547,24 @@
         }
 
         render() {
-            if (this.state.recordingID === null) {
-                return <RecordingList list={this.state.recordingList} />;
-            } else {
+            const dateSince = this.state.dateSince;
+            const dateUntil = this.state.dateUntil;
+
+            if(this.state.recordingID === null) {
                 return (
-                    <Recording
-                        recording={this.recordingMap[this.state.recordingID]}
-                    />
+                    <div>
+                    <RecordingList
+                        dateSince={dateSince} onDateSinceChange={this.handleDateSinceChange}
+                        dateUntil={dateUntil} onDateUntilChange={this.handleDateUntilChange}
+                        list={this.state.recordingList} />
+                    </div>
+                );
+            }
+            else {
```

Comment:
Please keep `else` and braces on the same line. It seems to be the prevailing convention in Cockpit.